### PR TITLE
15.0 mig account_cutoff_prepaid

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ivantodorovich** [comment]
CI is red, could you check?

Also please clean the commit history. as there are multiple commits with `
[MIG] account_cutoff_prepaid: Migration to 15.0` message. They should be squashed

File: account_cutoff_base/models/account_cutoff.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def _compute_total_cutoff(self):
         )
         mapped_data = {x["parent_id"][0]: x["cutoff_amount"] for x in rg_res}
         for cutoff in self:
-            cutoff.total_cutoff_amount = mapped_data.get(cutoff.id, 0)
+            cutoff.total_cutoff_amount += mapped_data.get(cutoff.id, 0)
```
Comment (yankinmax) [inline account_cutoff_base/models/account_cutoff.py:31]: why "+="? do we need to add amount to existing?

File: account_cutoff_base/models/account_cutoff.py [hunk 2]
```diff
@@ -353,7 +353,8 @@ def _get_mapping_dict(self):
         """return a dict with:
         key = ID of account,
         value = ID of cutoff_account"""
-        self.ensure_one()
+        for record in self:
+            record.ensure_one()
```
Comment (yankinmax) [inline account_cutoff_base/models/account_cutoff.py:357]: this is not needed, `self.ensure_one()` verifies that the current recorset holds a single record, on the model level

**Comment by leemannd** [comment]
Hello @juliette-blanc @OCA/accounting-maintainers This PR should be closed. The features have been merged into the module `account_cutoff_start_end_dates`

-> https://github.com/OCA/account-closing/issues/133#issue-606569879

**Review by leemannd — CHANGES_REQUESTED** [review]
This shouldn't be merged. Please check above comment.

The features of this module have been merged into another one.

**Pull request closed without merging**